homeassistant/components/statistics/sensor.py

```diff
@@ -186,6 +186,7 @@
 CONF_STATE_CHARACTERISTIC = "state_characteristic"
 CONF_SAMPLES_MAX_BUFFER_SIZE = "sampling_size"
 CONF_MAX_AGE = "max_age"
+CONF_PRESERVE_LAST_VAL = "preserve_last_val"
```

Comment:
There's already a configuration option related to samples: `sampling_size`. I think it would be better to use the term "sample" here too so it's clear they refer to the same thing, hence `preserve_last_sample` or something like that.

Reply:
Agreed, this is more consistent. Used `keep_last_sample`. Updated documentation PR as well.